### DISCUSSION TITLE
Contributions

### DISCUSSION THREAD

**Discussion by NrujalSharma** [elsa-workflows/elsa-core#3854] [General]
Hi @sfmskywalker & @jdevillard ,
Elsa v2 has a search facility for activities in the designer. v3 as of now didn't have that. I've created a [PR](https://github.com/elsa-workflows/elsa-core/pull/3853) regarding the same on the repo.

Since this is my first contribution to an open source project, I wasn't sure if I needed to follow some other procedure before raising the PR. Let me know if the PR needs some fixing.

Thanks.

**Comment by sfmskywalker** [comment]
Hi @NrujalSharma , I am pleased that your first open source contribution is with our project, well done!

There are no official procedures. Usually, we like to create an issue to describe the feature or bug, but it's not required.
And in this case, there was already an issue created for it (#3156) which means we are happy with your PR.

Thank you for your contribution!
